opm/simulators/timestepping/TimeStepControl.hpp

```diff
@@ -32,7 +33,8 @@ namespace Opm
       SimpleIterationCount,
       PID,
       PIDAndIterationCount,
-      HardCodedTimeStep
+      HardCodedTimeStep,
+      General3rdOrder
```

Comment:
I suggest adding a trailing comma here (`General3rdOrder`) to simplify future maintenance.